src/CurrencyAPI/Currency.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections.Generic;
+using System.Text;
+
+namespace ePiggyWeb.CurrencyAPI
+{
+    public class Currency
+    {
+        public string Name { get; set; }
+
+        public string Code { get; set; }
+
+        public IEnumerable<int> Symbol { get; set; }
+
+        public decimal Rate { get; set; }
+
+        public string GetSymbol()
+        {
+            var sb = new StringBuilder();
+
+            foreach (var symbol in Symbol)
+            {
+                sb.Append(char.ConvertFromUtf32(symbol));
```

Comment:
Why is it stored in such format?

Reply:
Well shortest answer is because that what I found on the internet, but I figure why the symbols are stored as an array of integers - it is because the currency symbols can contain unique symbols so it is easier to transfer them as their unicode representations, and an array because some currencies only use one symbol and some use more than one